webshare/share-url-noscheme-manual.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share with URL without a scheme</title>
```

Comment:
Should this test and a few of the following not be .https? Otherwise they'll just fail for the wrong reason?